rune/tools/rpm/README.md

```diff
@@ -0,0 +1,17 @@
+# rune rpm
+## Build Requirements
+- golang >= 1.14
+- protobuf >= 3
+- protobuf-compiler
+- libseccomp-devel
+
+``` shell
+cd "${path_to_inclavare_containers}"
+make rpm
+```
+
+## install
+``` shell
+cd /root/rpmbuild/RPMS/x86_64
+rpm -ivh ${rune-rpm}
+```
```

Comment:
How about enhancing your make rpm to guess current linux distribution? First get to know about the package manager (deb or rpm style), then get the package manager command to check the requirements and print the missing dependencies.